shap/plots/_image.py

```diff
@@ -5,6 +5,8 @@
 import string
 from typing import TYPE_CHECKING, Literal, cast
 
+import cv2
```

Comment:
@connortann @CloseChoice Can you confirm if you would like me to create the plots without using ```keras``` and ```cv2```? Also, since ```torchvision``` is a dependency we could simply just use it, right?